oci-umount.spec

```diff
@@ -54,6 +54,7 @@ make %{?_smp_mflags}
 /usr/share/containers/oci/hooks.d/oci-umount.json
 /usr/share/oci-umount/oci-umount.d
 %ghost /etc/oci-umount/oci-umount.d
+%config /usr/share/oci-umount/oci-umount-options.conf
```

Comment:
This is not config

  %ghost /etc/oci-umount/oci-umount-options.conf

Reply:
ok, so config is only used for files which can be edited by user and we don't want to overwrite these over upgrade. This one we don't want users to edit and even if they do, they will lose change over upgrade, so this is not a config. Agreed.

But this does not seem to be `ghost` either? ghost seems to be files which are not shipped with the package. But we are shipping this file with the package?

Reply:
I ghosted the /etc version not the /usr version.

Reply:
aha.. missed that.

Ok, so by making it `ghost`, we will remove this file when package is installed. And that probably is fine.